models/User.js

```diff
@@ -0,0 +1,20 @@
+// Importation de Mongoose
+const mongoose = require("mongoose");
+
+// Rajout d'un validateur comme plugin à notre schéma  (mongoose unique validator)
+//(pour pas avoir plusieurs utilisateurs avec la même adresse mail)
+// car "unique=true" inssufisant (possibles erreurs de Mongodb).
+const uniqueValidator = require("mongoose-unique-validator");
+
+// Création d'un schéma d'utilisateur
+const userSchema = mongoose.Schema({
+  email: { type: String, required: true, unique: true },
```

Comment:
Add error message to unique

Reply:
Do I need to add a "user.updateOne" and {runValidators: true} and a "then/catch" to update informations ?
I heard that Update validators are off by default.